가운데글자가져오기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    let answer;
+    let word = s.length;
+    
+    if (word % 2 === 1) {
+        answer = s[Math.floor(word / 2)];
```

Comment:
```suggestion
        return s[Math.floor(word / 2)];
```
조건 검사가 끝난후 바로 반환한다면 조건식 안에서 반환해도 좋을것같아요